tajo-core/src/main/java/org/apache/tajo/engine/query/TaskRequestImpl.java

```diff
@@ -35,113 +35,113 @@
 
 public class TaskRequestImpl implements TaskRequest {
 
-	private TaskAttemptId id;
-	private List<FragmentProto> fragments;
-	private String outputTable;
-	private boolean isUpdated;
-	private boolean clusteredOutput;
-	private PlanProto.LogicalNodeTree plan;     // logical node
-	private Boolean interQuery;
-	private List<FetchImpl> fetches;
-	private QueryContext queryContext;
-	private DataChannel dataChannel;
-	private Enforcer enforcer;
-	private String queryMasterHostAndPort;
-	
-	private TaskRequestProto proto = TaskRequestProto.getDefaultInstance();
-	private TaskRequestProto.Builder builder = null;
-	private boolean viaProto = false;
-	
-	public TaskRequestImpl() {
-		builder = TaskRequestProto.newBuilder();
-		this.id = null;
-		this.isUpdated = false;
-	}
-	
-	public TaskRequestImpl(TaskAttemptId id, List<FragmentProto> fragments,
-												 String outputTable, boolean clusteredOutput,
-												 PlanProto.LogicalNodeTree plan, QueryContext queryContext, DataChannel channel,
-												 Enforcer enforcer, String queryMasterHostAndPort) {
-		this();
-		this.set(id, fragments, outputTable, clusteredOutput, plan, queryContext, channel, enforcer, queryMasterHostAndPort);
-	}
-	
-	public TaskRequestImpl(TaskRequestProto proto) {
-		this.proto = proto;
-		viaProto = true;
-		id = null;
-		isUpdated = false;
-	}
-	
-	public void set(TaskAttemptId id, List<FragmentProto> fragments,
-			String outputTable, boolean clusteredOutput,
-			PlanProto.LogicalNodeTree plan, QueryContext queryContext,
-									DataChannel dataChannel, Enforcer enforcer, String queryMasterHostAndPort) {
-		this.id = id;
-		this.fragments = fragments;
-		this.outputTable = outputTable;
-		this.clusteredOutput = clusteredOutput;
-		this.plan = plan;
-		this.isUpdated = true;
+  private TaskAttemptId id;
```

Comment:
It is right fix. But, it should be resolved in another jira. Could you create new jira for it?